core/src/main/java/com/graphhopper/routing/weighting/FastestWeighting.java

```diff
@@ -39,12 +41,20 @@
     private final double headingPenalty;
     private final long headingPenaltyMillis;
     private final double maxSpeed;
+    private EnumEncodedValue<RoadAccess> roadAccessEnc = null;
+    private double roadAccessPenalty;
 
     public FastestWeighting(FlagEncoder encoder, PMap map) {
         super(encoder);
         headingPenalty = map.getDouble(Routing.HEADING_PENALTY, Routing.DEFAULT_HEADING_PENALTY);
         headingPenaltyMillis = Math.round(headingPenalty * 1000);
         maxSpeed = encoder.getMaxSpeed() / SPEED_CONV;
+
+        if (encoder.hasEncodedValue(RoadAccess.KEY)) {
+            // ensure that we do not need to change getMinWeight, i.e. road_access_delay is actual a penalty
```

Comment:
`road_access_delay` ? You probably mean `roadAccessPenalty` in the comment. And the import thing is its >=1 (which you enforce already).

Reply:
Maybe add a comment on the roadAccessPenalty about what it is supposed to do.

Reply:
> road_access_delay

ups, this was the name before ;) .. will correct it

> Maybe add a comment on the roadAccessPenalty about what it is supposed to do

Will do